src/semantic/src/definitions/collections/grid.less

```diff
@@ -1713,10 +1713,10 @@
 }
 
 /*-------------------
-      Stackable
+      Stackable @largestMobileScreen)
```

Comment:
What is this setting?

Reply:
it's done automatically by semantic cause of the settings above

Reply:
ok